### ISSUE TITLE
SDSS-R missing from filter set

### ISSUE THREAD

**Issue by profxj** [pypeit/PypeIt#845]
The `orig_build` appears to have a typo:

```
    par = fits.open('filter_curves_sdss.fits')
    for i in ['SDSS-U','SDSS-G','SDSS-I','SDSS-G','SDSS-Z']:
        wave, trans = par[i[-1]].data['wavelength'], par[i[-1]].data['respt']
        hdu = tohdu(wave,trans,i)
        hdulist.append(hdu)
```

This should be rebuilt to add in SDSS-R.

**Comment by feigewang** [comment]
@profxj I have fixed the fits file. Which branch should I use to import this fix?

**Comment by profxj** [comment]
Great!  I suggest you just make a new branch and simple PR.
That will be best.  Thanks.

**Comment by profxj** [comment]
Fixed in PR #858
